src/main/resources/templates/in-app/day-one/v1.0.0.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en" xmlns:th="http://www.thymeleaf.org">
+  <head>
+    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
+    <title>Day One</title>
+  </head>
+  <body>
+    <h1>In-App Notification</h1>
+    <h2>Subject</h2>
+    <th:block th:fragment="subject">Day One</th:block>
+    <h2>Content</h2>
+    <th:block th:fragment="content">
+      <p>
+        This notification pertains to the following programme
```

Comment:
I think we had better wording for this line?

Reply:
Just having a quick look at the existing notifications. We keep using `This notification pertains to the following programme...` for all in-app programme notifications. While the placement in-app use `For your placement starting (date) at (site) in (speciality name), please see the below information.`
Do you think it would be good if we change this to something like `For your programme (name) starting on (date), please see the below information.`?

Reply:
Best to check with Naz, but that would be my personal preference